src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,61 @@
+package homework_2.random_chars_table;
+
+import java.util.InputMismatchException;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RandomCharsTable {
+    public static Scanner scanner = new Scanner(System.in);
+    private static int length;
+    private static int width;
+
+    public static void main(String[] args) {
+        System.out.println("Enter valid length, width and strategy in format \"length width strategy\" ");
+        printRandomCharsTable();
+    }
+
+    public static void printRandomCharsTable() {
+        try {
+            length = scanner.nextInt();
+            width = scanner.nextInt();
+            String strategy = scanner.next();
+            char[][] table = new char[length][width];
+
+            Random random = new Random();
+
+            for (int i = 0; i < length; i++) {
+                for (int j = 0; j < width; j++) {
+                    table[i][j] = (char) (random.nextInt((90 - 65) + 1) + 65);
+                    System.out.print("|" + table[i][j] + "|");
+                }
+                System.out.println();
+            }
+            checkStratAndExecute(strategy, table);
+        } catch (InputMismatchException e) {
+            System.out.println("You entered invalid value, try again.");
+            System.exit(0);
```

Comment:
Bad way of exiting program. Also it seems you have memory leak, right? Where do you close scanner?